core-tests/src/test/java/org/mule/routing/UntilSuccessfulTestCase.java

```diff
@@ -133,6 +134,7 @@ public void testSuccessfulDeliveryAckExpression() throws Exception
         untilSuccessful.start();
 
         final MuleEvent testEvent = getTestEvent("test_data");
+        ((DefaultMuleMessage)testEvent.getMessage()).assertAccess(true);
```

Comment:
Is ugly to have to use a cast in order to assert the result of this test. Also, fix the code format